packages/xdg_directories/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [0.1.1] - Bump version to reduce dependencies on external libraries.
```

Comment:
Should this just say "Reduce dependencies on external libraries" - I assume that every entry here will be a version bump?

Reply:
Sure.